server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessageContentExtractor.java

```diff
@@ -67,12 +91,8 @@ private MessageContent parseMultipartMixed(Multipart multipart) throws IOExcepti
         List<Entity> parts = multipart.getBodyParts();
         if (! parts.isEmpty()) {
             Entity firstPart = parts.get(0);
```

Comment:
It is hard to find resources on Multipart handling. From my understanding we must look at first part, else that can mean that we are not able to read the image, and so no text content should be retrieved. Have you a reason to think otherwise?